main.py

```diff
@@ -1,29 +1,89 @@
 """
 This is an Add-On for DocumentCloud which wraps the tabula-py library
 https://github.com/chezou/tabula-py
-
-It attempts to convert tables from PDFs to CSVs
+It allows you to export tables from a set of PDFs into CSVs and
+download the resulting CSVs as a zip file.
 """
 
+import os
+import sys
+import zipfile
+import subprocess
+from urllib.parse import urlparse
+
+import requests
 import tabula
+
 from documentcloud.addon import AddOn
+import lootdl
+from lootdl import DROPBOX_URL, GDRIVE_URL, MEDIAFIRE_URL, WETRANSFER_URL
 
 
 class Tabula(AddOn):
     """A tabula Add-On for DocumentCloud"""
 
-    def main(self):
+    def fetch_files(self, url):
```

Comment:
If we are going to want to support cloud downloads for many add-ons, we should think about turning this code into a proper library and putting it on PyPi